addons/google_calendar/models/google_calendar.py

```diff
@@ -846,20 +830,20 @@ def update_events(self, cr, uid, lastSync=False, context=None):
                         if event.GG.status:
                             parent_event = {}
                             if not event_to_synchronize[base_event][0][1].OE.event_id:
-                                main_ev = att_obj.search_read(cr, uid, [('google_internal_event_id', '=', event.GG.event['id'].rsplit('_', 1)[0])], fields=['event_id'], context=context_novirtual)
+                                main_ev = CalendarAttendee.with_context(context_novirtual).search_read([('google_internal_event_id', '=', event.GG.event['id'].rsplit('_', 1)[0])], fields=['event_id'])
```

Comment:
can we use search with limit=1 instead of search_read ? because here only first record used by main_ev[0]